src/lib/components/HyperLink/HyperLink.js

```diff
@@ -0,0 +1,30 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const HyperLink = (props) => {
```

Comment:
Do we need to add how to use this comment in the documentation?

Reply:
@kahummer Sorry, I meant **component**. DO we have documentation for this component (what it does, how to import and use) Looks like it should be part of Menu/legend info Links documentation?